tests/test_pipelines_common.py

```diff
@@ -76,6 +81,35 @@ def get_dummy_inputs(self, device, seed=0):
             "See existing pipeline tests for reference."
         )
 
+    @property
+    def params(self) -> frozenset:
+        raise NotImplementedError(
+            "You need to set the attribute `params` in the child test class. "
+            "`params` are checked if they can be passed to `__call__`. "
+            "`params` are used for families of pipelines that have common parameters such as text to "
+            "image pipelines for prompts and prompt embedding overrides. `pipeline_params.py` provides "
+            "some common sets of parameters such as `TEXT_TO_IMAGE_PARAMS`, `IMAGE_VARIATION_PARAMS`, etc... "
+            "If your pipeline's set of arguments has minor changes from one of the common sets of arguments, "
+            "do not make modifications to the existing common sets of arguments. I.e. a text to image pipeline "
+            "with non-configurable height and width arguments should set the attribute as "
+            "`params = TEXT_TO_IMAGE_PARAMS - {'height', 'width'}`. "
+            "See existing pipeline tests for reference."
+        )
```

Comment:
`params` with error message/docs

Reply:
error message is a little bit confusing to me - tried to edit it but also not 100% sure if I understood correctly

```suggestion
    @property
    def params(self) -> frozenset:
        raise NotImplementedError(
            "You need to set the attribute `params` in the child test class. "
            "`params` are used to check if all required parameters are present in `__call__`'s signature."
            " You can set `params` using one of the common set of parameters defined in`pipeline_params.py`"
            " e.g., `TEXT_TO_IMAGE_PARAMS` defines the common parameters used in text to  "
            "image pipelines, including prompts and prompt embedding overrides."
            "If your pipeline's set of arguments has minor changes from one of the common sets of arguments, "
            "do not make modifications to the existing common sets of arguments. I.e. a text to image pipeline "
            "with non-configurable height and width arguments should set the attribute as "
            "`params = TEXT_TO_IMAGE_PARAMS - {'height', 'width'}`. "
            "See existing pipeline tests for reference."
        )
```

Reply:
nice! edits look good, made one small change https://github.com/huggingface/diffusers/pull/2502/commits/86ec20922d7c740c0e463bb1384b3b8513a3b5e2